### PR TITLE
Add region tags

### PR DESCRIPTION
**Pull request by PierrickVoulet**

### CODE REVIEW

**Review by BenRKarl — APPROVED** [review]

File: examples/advanced_operations/add_ad_customizer.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def main(client, customer_id, ad_group_ids):
     )
 
 
+# [START add_ad_customizer]
```
Comment (wihl) [inline examples/advanced_operations/add_ad_customizer.py:69]: move to line 30

File: examples/advanced_operations/add_smart_display_ad.py [hunk 2]
```diff
@@ -179,8 +182,10 @@ def _create_smart_display_campaign(client, customer_id, budget_resource_name):
         return campaign_response.results[0].resource_name
     except GoogleAdsException as ex:
         _handle_google_ads_exception(ex)
+        # [END add_smart_display_ad_4]
 
 
+# [START add_smart_display_ad]
```
Comment (wihl) [inline examples/advanced_operations/add_smart_display_ad.py:188]: move to line 50

File: examples/advanced_operations/add_smart_display_ad.py [hunk 3]
```diff
@@ -198,8 +203,10 @@ def _create_ad_group(client, customer_id, campaign_resource_name):
         return ad_group_response.results[0].resource_name
     except GoogleAdsException as ex:
         _handle_google_ads_exception(ex)
+        # [END add_smart_display_ad]
```
Comment (wihl) [inline examples/advanced_operations/add_smart_display_ad.py:206]: move to line 119

File: examples/advanced_operations/add_smart_display_ad.py [hunk 4]
```diff
@@ -119,6 +119,7 @@ def main(
     )
 
 
+# [START add_smart_display_ad_3]
 def _create_budget(client, customer_id):
```
Comment (wihl) [inline examples/advanced_operations/add_smart_display_ad.py:123]: should be add_smart_display_ad_1

File: examples/advanced_operations/add_smart_display_ad.py [hunk 5]
```diff
@@ -141,8 +142,10 @@ def _create_budget(client, customer_id):
         return campaign_budget_response.results[0].resource_name
     except GoogleAdsException as ex:
         _handle_google_ads_exception(ex)
+        # [END add_smart_display_ad_3]
```
Comment (wihl) [inline examples/advanced_operations/add_smart_display_ad.py:145]: should be _ad_1

File: examples/advanced_operations/use_portfolio_bidding_strategy.py [hunk 6]
```diff
@@ -31,6 +31,7 @@ def main(client, customer_id):
     )
     campaign_service = client.get_service("CampaignService", version="v6")
 
+    # [START use_portfolio_bidding_strategy]
```
Comment (wihl) [inline examples/advanced_operations/use_portfolio_bidding_strategy.py:34]: move to line 24

File: examples/advanced_operations/use_portfolio_bidding_strategy.py [hunk 7]
```diff
@@ -59,11 +60,13 @@ def main(client, customer_id):
                 for field_path_element in error.location.field_path_elements:
                     print("\t\tOn field: %s" % field_path_element.field_name)
         sys.exit(1)
+        # [END use_portfolio_bidding_strategy]
```
Comment (wihl) [inline examples/advanced_operations/use_portfolio_bidding_strategy.py:63]: this file also needs a number of fixups

File: examples/basic_operations/add_campaigns.py [hunk 8]
```diff
@@ -62,7 +63,9 @@ def main(client, customer_id):
                 for field_path_element in error.location.field_path_elements:
                     print("\t\tOn field: %s" % field_path_element.field_name)
         sys.exit(1)
+        # [END add_campaigns_1]
 
+    # [START add_campaigns]
```
Comment (wihl) [inline examples/basic_operations/add_campaigns.py:68]: move to line 32

File: examples/basic_operations/add_campaigns.py [hunk 9]
```diff
@@ -35,6 +35,7 @@ def main(client, customer_id):
     )
     campaign_service = client.get_service("CampaignService", version="v6")
 
+    # [START add_campaigns_1]
```
Comment (wihl) [inline examples/basic_operations/add_campaigns.py:38]: remove

File: examples/basic_operations/add_campaigns.py [hunk 10]
```diff
@@ -62,7 +63,9 @@ def main(client, customer_id):
                 for field_path_element in error.location.field_path_elements:
                     print("\t\tOn field: %s" % field_path_element.field_name)
         sys.exit(1)
+        # [END add_campaigns_1]
```
Comment (wihl) [inline examples/basic_operations/add_campaigns.py:66]: remove

File: examples/basic_operations/add_campaigns.py [hunk 11]
```diff
@@ -85,6 +88,7 @@ def main(client, customer_id):
     campaign.network_settings.target_search_network = True
     campaign.network_settings.target_content_network = False
     campaign.network_settings.target_partner_search_network = False
+    # [END add_campaigns]
```
Comment (wihl) [inline examples/basic_operations/add_campaigns.py:91]: move to line 118

File: examples/campaign_management/add_complete_campaigns_using_batch_job.py [hunk 12]
```diff
@@ -129,6 +129,7 @@ def _create_batch_job_operation(client):
     return batch_job_operation
 
 
+# [START add_complete_campaigns_using_batch_job]
```
Comment (wihl) [inline examples/campaign_management/add_complete_campaigns_using_batch_job.py:132]: this file needs a lot of fixups

File: examples/campaign_management/get_campaigns_by_label.py [hunk 13]
```diff
@@ -66,6 +67,7 @@ def main(client, customer_id, label_id, page_size):
             )
     except GoogleAdsException as ex:
         print_error_and_exit_process(ex)
+        # [END get_campaigns_by_label]
```
Comment (wihl) [inline examples/campaign_management/get_campaigns_by_label.py:70]: move to line 88

File: examples/error_handling/handle_expanded_text_ad_policy_violations.py [hunk 14]
```diff
@@ -118,6 +118,7 @@ def _create_expanded_text_ad(
     return ad_group_ad_operation, ignorable_policy_topics
 
 
+# [START handle_expanded_text_ad_policy_violations]
```
Comment (wihl) [inline examples/error_handling/handle_expanded_text_ad_policy_violations.py:121]: this file needs a lot of fixups

File: examples/error_handling/handle_keyword_policy_violations.py [hunk 15]
```diff
@@ -134,6 +134,7 @@ def _create_keyword_criterion(
     return None, ad_group_criterion_operation
 
 
+# [START handle_keyword_policy_violations]
```
Comment (wihl) [inline examples/error_handling/handle_keyword_policy_violations.py:137]: move to line 35

File: examples/error_handling/handle_keyword_policy_violations.py [hunk 16]
```diff
@@ -191,8 +192,10 @@ def _fetch_exempt_policy_violation_keys(google_ads_exception):
             raise google_ads_exception
 
     return exempt_policy_violation_keys
+    # [END handle_keyword_policy_violations]
 
 
+# [START handle_keyword_policy_violations_1]
```
Comment (wihl) [inline examples/error_handling/handle_keyword_policy_violations.py:198]: are you sure you want this here?

File: examples/error_handling/handle_partial_failure.py [hunk 17]
```diff
@@ -56,6 +56,7 @@ def main(client, customer_id, campaign_id):
         print_results(client, ad_group_response)
 
 
+# [START handle_partial_failure]
```
Comment (wihl) [inline examples/error_handling/handle_partial_failure.py:59]: this file needs a lot of fixups

File: examples/extensions/add_sitelinks.py [hunk 18]
```diff
@@ -31,6 +31,7 @@
 _date_format = "%Y-%m-%d %H:%M:%S"
 
 
+# [START add_sitelinks_1]
```
Comment (wihl) [inline examples/extensions/add_sitelinks.py:34]: this file needs a lot of fixups

File: examples/hotel_ads/add_hotel_ad.py [hunk 19]
```diff
@@ -89,6 +89,7 @@ def add_budget(client, customer_id):
     return budget_resource_name
 
 
+# [START add_hotel_ad_1]
```
Comment (wihl) [inline examples/hotel_ads/add_hotel_ad.py:92]: this file needs a lot of fixups

File: examples/planning/forecast_reach.py [hunk 20]
```diff
@@ -83,8 +84,10 @@ def _show_plannable_products(client, location_id):
         print("Devices:")
         for device in product_metadata.plannable_targeting.devices:
             print(f"\t- {device_enum.Device.Name(device.type)}")
+            # [END forecast_reach_2]
 
 
+# [START forecast_reach]
```
Comment (wihl) [inline examples/planning/forecast_reach.py:90]: this file needs a lot of fixups

File: examples/remarketing/add_remarketing_action.py [hunk 21]
```diff
@@ -40,6 +40,7 @@ def main(client, customer_id, page_size):
     _print_remarketing_action_attributes(client, queried_remarketing_action)
 
 
+# [START add_remarketing_action_1]
```
Comment (wihl) [inline examples/remarketing/add_remarketing_action.py:43]: this file needs a lot of fixups

File: examples/shopping_ads/add_shopping_product_ad.py [hunk 22]
```diff
@@ -106,6 +106,7 @@ def add_campaign_budget(client, customer_id):
     return budget_resource_name
 
 
+# [START add_shopping_product_ad]
```
Comment (wihl) [inline examples/shopping_ads/add_shopping_product_ad.py:109]: this file needs a lot of fixups

File: examples/shopping_ads/approve_merchant_center_link.py [hunk 23]
```diff
@@ -45,6 +45,7 @@ def main(client, customer_id, merchant_center_account_id):
     )
 
     try:
+        # [START approve_merchant_center_link]
```
Comment (wihl) [inline examples/shopping_ads/approve_merchant_center_link.py:48]: this file needs a lot of fixups

File: examples/targeting/add_campaign_targeting_criteria.py [hunk 24]
```diff
@@ -55,6 +55,7 @@ def main(client, customer_id, campaign_id, keyword_text, location_id):
         print(f'Added campaign criterion "{result.resource_name}".')
 
 
+# [START add_campaign_targeting_criteria]
```
Comment (wihl) [inline examples/targeting/add_campaign_targeting_criteria.py:58]: this file needs a lot of fixups

**Review by wihl — COMMENTED** [review]
many changes needed

**Review by wihl — APPROVED** [review]
LGTM as is with content issues to be addressed in future PRs

**Pull request merged**